fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -837,9 +833,8 @@ public void initialize(String[] args) throws Exception {
         // 6. start task cleaner thread
         createTaskCleaner();
 
-        // 7. start state listener thread
-        createStateListener();
-        listener.start();
+        // 7. set meta context to state change executor
+        StateChangeExecutor.getInstance().setMetaContext(metaContext);
```

Comment:
Better to move to `StarRocksFE.java` and execute it before `StateChangeExecutor.start()`.

Reply:
done